docs/releases/1.10.txt

```diff
@@ -301,7 +301,7 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* Added methods :meth:`~django.test.testcases.SimpleTestCase.assertFormValid` and :meth:`~django.test.testcases.SimpleTestCase.assertFormNotValid`.
```

Comment:
Thanks @timgraham that eliminates the warning locally. Committed the change, rebased to squash last commit and pushed again. Thanks!